src/runner/main.cpp

```diff
@@ -82,11 +82,12 @@ void open_menu_from_another_instance(std::optional<std::string> settings_window)
 {
     const HWND hwnd_main = FindWindowW(L"PToyTrayIconWindow", nullptr);
     LPARAM msg = static_cast<LPARAM>(ESettingsWindowNames::Overview);
-    if (settings_window.has_value())
+    if (settings_window.has_value() && settings_window.value() != "")

```

Comment:
This seemed to fix a problem I had relating to #8277 however `std::optional<std::string>` should still return 0 when the value is `""`. Need to investigate further as testing elsewhere shows it returns 0.

Reply:
PR #25753 contains this specific fix to resolve #8277 as it is another issue outside the scope of this PR.